src/components/modules/banners/digitalSubscriptions/DigitalSubscriptionsBanner.stories.tsx

```diff
@@ -28,7 +28,7 @@ const tracking: BannerTracking = {
 
 export const defaultStory = (): ReactElement => {
     const content: BannerContent = {
-        heading: text('heading', 'Start a digital subscription today'),
+        heading: text('heading', "You've read %%ARTICLE_COUNT%% articles in the last year."),
```

Comment:
Now that you're using templates in your copy it's worth using the `containsNonArticleCountPlaceholder` function in your components, to avoid rendering a banner if there's a typo in the placeholder.
https://github.com/guardian/support-dotcom-components/blob/1bc5c6f2ddcc804879cd2d7e54dc8078c85821dd/src/lib/placeholders.ts#L23

Reply:
Ah - thank you!

Reply:
Hey @tomrf1 I've made some changes based on your comments - is this what you meant?

Reply:
Yep looks good. We didn't bother with fallback copy for our banners, it just renders nothing!
The tool is supposed to catch invalid templates, but it's always best to be safe and catch it at the point of rendering as well!
This stuff has gone wrong in the past...

Reply:
Cool, thanks Tom 👍